dsc/cli.py

```diff
@@ -6,27 +6,117 @@
 import click
 
 from dsc.config import Config
+from dsc.utilities import (
+    build_bitstream_dict,
+    match_bitstreams_to_item_identifiers,
+    match_item_identifiers_to_bitstreams,
+)
+from dsc.workflows.base import BaseWorkflow
 
 logger = logging.getLogger(__name__)
 CONFIG = Config()
 
 
-@click.command()
+@click.group()
+@click.pass_context
+@click.option(
+    "-w",
+    "--workflow_name",
+    help="The workflow to use for the batch of DSpace submissions",
+    required=True,
+)
+@click.option(
+    "-b",
+    "--batch_id",
+    help="The S3 prefix for the batch of DSpace submissions",
+    required=True,
+)
+@click.option(
+    "-c",
+    "--collection_handle",
+    help="The handle of the DSpace collection to which the batch will be submitted",
+    required=True,
+)
```

Comment:
Can we use dashes to separate multi-word command line arguments to follow Unix/Linux conventions? 🤔

Reply:
Good call, updated!